### ISSUE TITLE
Footer Warning

### ISSUE THREAD

**Issue by rmccar** [ONSdigital/design-system#1100]
Use this issue to discuss the [Footer Warning](https://ons-design-system.netlify.com/footer-warning) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by trevorsaint** [comment]
I would be tempted to make the link black (dark version we use in the footer). 

**Reason**: Accessibility enhancement WCAG AAA
<img width="946" alt="Screenshot 2020-10-16 at 11 16 20" src="https://user-images.githubusercontent.com/4989027/96246804-12883600-0fa1-11eb-8ccd-cec5ae69fee0.png">

**Comment by rmccar** [comment]
> I would be tempted to make the link black (dark version we use in the footer).
> 
> **Reason**: Accessibility enhancement WCAG AA to WCAG AAA
> <img alt="Screenshot 2020-10-16 at 11 16 20" width="946" src="https://user-images.githubusercontent.com/4989027/96246804-12883600-0fa1-11eb-8ccd-cec5ae69fee0.png">

Im moving this to the footer component. Ill add this change in
